packages/histoire-controls/src/components/checkbox/HstCheckbox.vue

```diff
@@ -14,7 +14,7 @@ const props = defineProps<{
 }>()
 
 const emit = defineEmits({
-  'update:modelValue': (newValue: boolean) => true,
+  'update:modelValue': (_newValue: boolean) => true,
```

Comment:
Why this change?

Reply:
Linter was complaining, newValue is not being used, is just returning true.

We probably could get rid of the param but not sure why was there tho.

Reply:
It's here for type inference, but maybe you can turn it to the Generic version of `defineEmits`